fendermint/testing/materializer/src/docker/node.rs

```diff
@@ -205,13 +205,29 @@ impl DockerNode {
 
         export_file(keys_dir.join(COMETBFT_NODE_ID), cometbft_node_id)?;
 
+        fendermint_runner
+            .run_cmd(
+                "genesis \
+                    --genesis-file /fendermint/genesis.json \
+                    ipc \
+                        seal-genesis \
+                        --builtin-actors-path /fendermint/bundle.car \
+                        --custom-actors-path /fendermint/custom_actors_bundle.car \
+                        --artifacts-path /fendermint/contracts \
+                        --output-path /cometbft/config/sealed.json \
+                    ",
+            )
+            .await
+            .context("failed to seal genesis state")?;
+
         // Convert fendermint genesis to cometbft.
         fendermint_runner
             .run_cmd(
                 "genesis \
                     --genesis-file /fendermint/genesis.json \
                     into-tendermint \
                     --out /cometbft/config/genesis.json \
+                    --app-state /cometbft/config/sealed.json \
```

Comment:
Why both an out and an `app-state`?

Reply:
The `app-state` is practically the genesis state, but the terminology cometbft is using is `app-state`, just took that term here.